include/crm/common/output_internal.h

```diff
@@ -860,6 +860,7 @@ void pcmk__output_and_clear_error(GError *error, pcmk__output_t *out);
 
 int pcmk__xml_output_new(pcmk__output_t **out, xmlNodePtr *xml);
 void pcmk__xml_output_finish(pcmk__output_t *out, xmlNodePtr *xml);
+pcmk__output_t *pcmk__new_logger(void);
```

Comment:
For parallelism, it could be worth changing this function's prototype to `int pcmk__log_output_new(pcmk__output_t **out)`.

Reply:
I guess that's better than going the opposite direction with pcmk__new_xmler() :)